backend/module/eCampCore/src/Hydrator/UserHydrator.php

```diff
@@ -21,7 +21,8 @@ public function extract($object) {
         return [
             'id' => $user->getId(),
             'username' => $user->getUsername(),
-            'mail' => $user->getTrustedMailAddress()
+            'mail' => $user->getTrustedMailAddress(),
+            'role' => $user->getRole()
```

Comment:
Du gibst hier die Rolle des Users im Gesamtsystem aus (admin, user, etc.).

Wenn ich #342 lese, wollten wir hier aber eigentlich die Rolle des aktuell authentifizierten Users in Relation mit diesem User ausgeben (z.B. it's me). Kann man aber durchaus nochmals in Frage stellen, ob das was bringt.

Reply:
Zeige nun beides an:
relation: me/known/unrelated
role:member/admin